### PR TITLE
Added links for Spark, Mapr to MapR

### PR DESCRIPTION
**Pull request by Brijeshrpatel9**

### CODE REVIEW

File: BigData/BigDataTutorial.md [hunk 1]
```diff
@@ -9,12 +9,15 @@
...
+- [MapR Tutotial](http://learn.mapr.com/) - One of the Best Tutorial For Big Data and Hadoop
 - [Big Data Projects](https://www.acquia.com/examples-big-data-projects) - Big Data Project Ideas
 - [HDFS Architechture](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html) - Hadoop Distributed File System Arch.
-- [Mapr  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
+- [MapR  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
 - [Udacity Courses](https://www.udacity.com/courses/data-science) - Udacity MOOC Data Science Courses
 - [Coursera Big Data Specialization](https://www.coursera.org/specializations/big-data) - Coursera Big Data Course By University of California,San Diego
 - [Cloudera Hadoop Certification](http://www.cloudera.com/training/certification.html) 
-- [Mapr Hadoop Certification](https://www.mapr.com/services/mapr-academy/certified-hadoop-developer)
+- [MapR Hadoop Certification](https://www.mapr.com/services/mapr-academy/certified-hadoop-developer)
```
Comment (PareshMayani) [inline BigData/BigDataTutorial.md]: Certification should be part of "Useful links". For reference, check the android directory https://github.com/GDGAhmedabad/Learning-Resources/tree/master/android

File: BigData/BigDataTutorial.md [hunk 2]
```diff
@@ -9,12 +9,15 @@
 - [Official Apache Hadoop Site](https://hadoop.apache.org) - Apache Hadoop Site
 - [edureka Courses](https://www.youtube.com/user/edurekaIN/playlists?sort=dd&shelf_id=6&view=50) - Edureka's All Youtube Videos
 - [Online Competition For Data Science](https://www.kaggle.com/) - Kaggle
-- [Mapr Tutotial](http://learn.mapr.com/) - One of the Best Tutorial For Big Data and Hadoop
+- [MapR Tutotial](http://learn.mapr.com/) - One of the Best Tutorial For Big Data and Hadoop
 - [Big Data Projects](https://www.acquia.com/examples-big-data-projects) - Big Data Project Ideas
 - [HDFS Architechture](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html) - Hadoop Distributed File System Arch.
-- [Mapr  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
+- [MapR  Papers](https://www.mapr.com/resources/white-papers) - Hadoop White Papers
```
Comment (PareshMayani) [inline BigData/BigDataTutorial.md]: Again this should be part of "Useful links" section!

Comment (Brijeshrpatel9) [inline BigData/BigDataTutorial.md]: I didn't add these links. However, "Big Data" folder doesn't have folder structure. That is why I included all the links in one.

Comment (PareshMayani) [inline BigData/BigDataTutorial.md]: You can include new files but please check about files created in android directory!

**Comment by PareshMayani** [comment]
@Brijeshrpatel9 are you going to make changes and send commits for this PR?

**Comment by Brijeshrpatel9** [comment]
Closing this PR. I will make some modification and send another PR in May.

**Pull request closed without merging**